go/useeioapi/years.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"net/http"
+	"path/filepath"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+// Year describes a year.
+type Year struct {
+	ID          string `json:"id"`
+	Index       int    `json:"index"`
+}
+
+// ReadSectors reads the years from the CSV file in the data folder. It
```

Comment:
comment should start with `ReadYears`

Reply:
Fixed in this commit
https://github.com/USEPA/USEEIO_API/commit/094845d29e00ac39071b8da6e8976a4c2a4a6168